### PR TITLE
ocp-13065-upgrade-auto

### PR DESCRIPTION
**Pull request by sunilcio**
ocp-13065-upgrade-auto

### CODE REVIEW

**Comment by sunilcio** [comment]
@jhou1 @weinliu @lyman9966 can you please review. I still need to execute the test on env tomorrow as flexy today was failing due to quay outage.

File: features/upgrade/node/scc.feature [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  # @author sunilc@redhat.com
+  # @case_id OCP-13065
+  @upgrade-check
+  @users=upuser1
+  @admin
+  Scenario: Upgrade - Make sure seccomp part of SCC policy is kept after upgrade 
+    Given I switch to the first user
+    When I use the "node-upgrade-scc" project    
+    When I run the :get admin command with:
+      | resource       | scc                                            |
```
Comment (pruan-rht) [inline features/upgrade/node/scc.feature:25]: extra spaces before the ending `|`

File: features/upgrade/node/scc.feature [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      | f | <%= BushSlicer::HOME %>/features/tierN/testdata/node/scc.yaml |
+    Then the step should succeed
+
+  # @author sunilc@redhat.com
+  # @case_id OCP-13065
+  @upgrade-check
+  @users=upuser1
+  @admin
+  Scenario: Upgrade - Make sure seccomp part of SCC policy is kept after upgrade 
+    Given I switch to the first user
```
Comment (weinliu) [inline features/upgrade/node/scc.feature:22]: In the prepare steps, I see admin user used, I'm not sure if this should be identical. Other pars LGTM

**Comment by sunilcio** [comment]
Mistakenly created https://github.com/openshift/verification-tests/pull/1166 instead of updating this. Closing this one.

**Pull request closed without merging**